DependencyInjection/SuluRedirectExtension.php

```diff
@@ -55,12 +56,39 @@ public function prepend(ContainerBuilder $container)
                     'exception' => [
                         'codes' => [
                             RedirectRouteNotUniqueException::class => 409,
+                            RedirectRouteNotFoundException::class => 404,
                             EntityNotFoundException::class => 404,
                         ],
                     ],
                 ]
             );
         }
+
+        if ($container->hasExtension('sulu_admin')) {
+            $container->prependExtensionConfig(
+                'sulu_admin',
+                [
+                    'lists' => [
+                        'directories' => [
+                            __DIR__ . '/../Resources/config/lists',
+                        ],
+                    ],
+                    'forms' => [
+                        'directories' => [
+                            __DIR__ . '/../Resources/config/forms',
+                        ],
+                    ],
+                    'resources' => [
+                        'redirect_routes' => [
+                            'routes' => [
+                                'list' => 'get_redirect-routes',
```

Comment:
we should add a route prefix here:

```
sulu_redirect.get_redirect-routes
```

can be configured in the route config with:

```
...:
    type: rest
    ...
    name_prefix: sulu_redirect.
```